github/orgs_members.go

```diff
@@ -270,3 +270,33 @@ func (s *OrganizationsService) RemoveOrgMembership(user, org string) (*Response,
 
 	return s.client.Do(req, nil)
 }
+
+//ListPendingOrgInvitations returns a list of contains a role field which refers to the
+//Organization Invitation role and will be one of the following values:
+//direct_member, admin, billing_manager, hiring_manager, or reinstate.
+//If the invitee is not a GitHub member, the login field in the return hash will be null.
```

Comment:
This documentation is hard to read and strangely detailed (it refers to specifics of the returned values inside `Invitation` struct).

We should either omit it altogether, just say `// ListPendingOrgInvitations returns a list of pending organization invitations.`, or update it to be specific to go-github. The returned thing is `[]Invitation`, a slice of invitations, not a hash. There's no null, it's nil. Etc.

It would make sense to me to move the documentation of `role` to the `Invitation` struct itself, not here. Unless it's specific to this method, is it?